NIPTool/commands/load/base.py

```diff
@@ -10,6 +10,7 @@
 
```

Comment:
The import lines up to line 7 of this file are not used.

Reply:
😣